etc/openstackquickstartrc

```diff
@@ -29,6 +29,9 @@ with_horizon=yes
 # Setup Manila
 with_manila=yes
 
+# Setup Magnum
+with_magnum=yes
```

Comment:
oh I'm fine with adding a change to qa_openstack.sh to enable it.. I'm just talking about the usecase when openstack-quickstart is being used outside qa_openstack.sh.